R/setMBOControlInfill.R

```diff
@@ -164,34 +136,12 @@ setMBOControlInfill = function(control,
 
   assertClass(control, "MBOControl")
 
-  control$infill.crit = as.character(coalesce(crit, control$infill.crit, "mean"))
-  assertChoice(control$infill.crit, choices = getSupportedInfillCritFunctions())
+  control$infill.crit = coalesce(crit, control$infill.crit, makeMBOInfillCriterionMeanResponse())
+  #assertChoice(control$infill.crit, choices = getSupportedInfillCritFunctions())
 
```

Comment:
documentation is missing, it still states that the user should supply a character string